### PR TITLE
Add link to roadmap issues

### PR DESCRIPTION
**Pull request by davorrunje**
## Why are these changes needed?

This PR resolves an item in https://github.com/ag2ai/ag2/issues/40

## Checks

- [x] I've included any doc changes needed for https://ag2ai.github.io/ag2/. See https://ag2ai.github.io/ag2/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -252,6 +252,8 @@ In addition, you can find:
 
 - [Contributing guide](https://ag2ai.github.io/ag2/docs/Contribute)
 
+- [Roadmap](https://ag2.ai/#roadmap) and [roadmap issues](https://github.com/ag2ai/ag2/issues?q=is%3Aopen+is%3Aissue+label%3Aroadmap)
```
Comment (Hk669) [inline README.md:255]:
```suggestion
- [Roadmap](https://ag2.ai/#roadmap) and [Roadmap issues](https://github.com/ag2ai/ag2/issues?q=is%3Aopen+is%3Aissue+label%3Aroadmap)
```

**Comment by Hk669** [comment]
can you verify the checks, i can see something is unusual with the message related to "signing the commit". im not sure how to fix it. thanks for the PR  @davorrunje

**Comment by davorrunje** [comment]
> can you verify the checks, i can see something is unusual with the message related to "signing the commit". im not sure how to fix it. thanks for the PR @davorrunje
> 
> cc @sonichi can you help.

Fixed. Thanx for the heads up 👍

File: README.md [hunk 2]
```diff
@@ -252,6 +252,8 @@ In addition, you can find:
 
 - [Contributing guide](https://ag2ai.github.io/ag2/docs/Contribute)
 
+- [Roadmap](https://ag2.ai/#roadmap) and [Roadmap issues](https://github.com/ag2ai/ag2/issues?q=is%3Aopen+is%3Aissue+label%3Aroadmap)
+
```
Comment (Hk669) [inline README.md:256]:
```suggestion
- [Roadmap](https://ag2.ai/#roadmap) and [Roadmap Issues](https://github.com/ag2ai/ag2/issues?q=is%3Aopen+is%3Aissue+label%3Aroadmap)

```

**Review by Hk669 — APPROVED** [review]
Looks good to me. Thanks @davorrunje

**Pull request merged**